supported-services.md

```
@@ -0,0 +1,87 @@
+| ❌ [Deepl](https://developers.deepl.com/docs) |     | Very low requests per second |
+| ❌ elia |     | Very slow |
+| ❌ hujiang |     | Very low requests per second |
+| ❌ itranslate |     | Very low requests per second |
+| ❌ languageWire |     | Very low requests per second |
+| ❌ [Linguee](https://www.linguee.com/) |     | Very low requests per second |
+| ❌ [Mymemory](https://mymemory.translated.net/) |     | Very low request limit |
+| ❌ [OpenNMT](https://opennmt.net/) |     | Only for full texts |
+| ❌ [Papago](https://papago.naver.com/) |     | Very slow, mainly for Korean |
+| ❌ [QcriTranslator](https://mt.qcri.org/api/) |     | Obligatory registration failed |
+| ❌ qqTranSmart |     | Very slow |
+| ❌ [Sogou](# "fanyi.sogou.com/text") |     | Very slow |
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)